marge/job.py

```diff
@@ -426,14 +426,14 @@ def synchronize_using_gitlab_rebase(self, merge_request, verify_expected_sha=Tru
             if branch.protected:
                 raise CannotMerge("Sorry, I can't modify protected branches!") from err
             raise
-        else:
-            if verify_expected_sha and merge_request.sha != expected_sha:
-                raise GitLabRebaseResultMismatch(
-                    gitlab_sha=merge_request.sha,
-                    expected_sha=expected_sha,
-                )
 
-            return merge_request.sha
+        if verify_expected_sha and merge_request.sha != expected_sha:
+            raise GitLabRebaseResultMismatch(
+                gitlab_sha=merge_request.sha,
+                expected_sha=expected_sha,
+            )
+
+        return merge_request.sha
```

Comment:
Hmm, what does the old indentation even mean in this case? :thinking: `try` with `else`...?

Reply:
https://www.geeksforgeeks.org/try-except-else-and-finally-in-python/